fastify.js

```diff
@@ -643,6 +654,24 @@ function build (options) {
     this._errorHandler = func
     return this
   }
+
+  function getSchema (keyRef) {
+    return ajv.getSchema(keyRef)
```

Comment:
my main concern is that we are removing the possibility to use `Joy` or other json schema validators through `setSchemaCompiler`.

Reply:
Hmm. I think I see what you're thinking - internally I still use schemaCompiler (all the tests using Joi are passing). I'll refactor this method to use that one as well.

Reply:
great!